src/features/Respondents/Respondents.tsx

```diff
@@ -104,12 +109,43 @@ export const Respondents = () => {
         filterRows(secretId, searchValue) || filterRows(nickname, searchValue),
     );
 
+  const chosenRespondentsItems =
+    respondentsDataIndex || respondentsDataIndex === 0
```

Comment:
usersData?.items[respondentsDataIndex] you can use just

Reply:
I cannot, because: "Type 'null' cannot be used as an index type".

Reply:
(respondentsDataIndex || respondentsDataIndex === 0) && usersData?.items[respondentsDataIndex]

Reply:
When I have this error: "Argument of type 'false | 0 | Users | undefined' is not assignable to parameter of type 'Users | undefined'.   Type 'boolean' is not assignable to type 'Users'."

Reply:
ok keep it